src/structures/Channel.js

```diff
@@ -86,17 +86,18 @@ class Channel extends Base {
 
   /**
    * Deletes this channel.
+   * @param {string} [reason=undefined] The reason for the channel being deleted.
    * @returns {Promise<Channel>}
    * @example
    * // Delete the channel
    * channel.delete()
    *   .then(console.log)
    *   .catch(console.error);
    */
-  delete() {
+  delete(reason = undefined) {
```

Comment:
Do you need to set default as `undefined`? I mean if there is no `reason`, it would be automatically `undefined`

Reply:
Yeah... I think my inner typescript dev came out, lol